pkg/sysinfo/sysinfo_linux.go

```diff
@@ -267,8 +267,6 @@ func applyDevicesCgroupInfo(info *SysInfo) {
 // applyNetworkingInfo adds networking information to the info.
 func applyNetworkingInfo(info *SysInfo) {
 	info.IPv4ForwardingDisabled = !readProcBool("/proc/sys/net/ipv4/ip_forward")
-	info.BridgeNFCallIPTablesDisabled = !readProcBool("/proc/sys/net/bridge/bridge-nf-call-iptables")
-	info.BridgeNFCallIP6TablesDisabled = !readProcBool("/proc/sys/net/bridge/bridge-nf-call-ip6tables")
```

Comment:
FWIW; this stops setting these fields, regardless of API version. I didn't update docs for older API versions, but given how we used this information thought it was OK to just stop setting it (worst case scenario, we add these lines back, and reset it in old API versions, but I highly doubt anyone outside of the daemon itself uses this information.